setup/gcp/scripts/run_forseti.sh

```diff
@@ -55,3 +55,10 @@ sleep 10s
 echo "Running Forseti notifier."
 forseti notifier run
 echo "Finished running Forseti notifier."
+sleep 10s
+
+# Clean up the model tables
+# Get model_id
+MODEL_ID=$(forseti model get ${MODEL_NAME} | python model_reader.py)
```

Comment:
Hmmm, this scrapping of the output seems something that can be improved with either a service method that can give you the model handle if you give it the model name. or the delete method should be able to delete by the model name (since it is a primary key).  Please open an issue or a TODO for future improvement.